jfrog-applications/jfrog-cli/cli-command-summaries.md

```diff
@@ -0,0 +1,150 @@
+# CLI Command Summaries
+
+## Overview
+
+The command summaries feature enables the recording of JFrog CLI command outputs into the local file system.
+This functionality can be used to generate a summary of the executed CLI command.
```

Comment:
This functionality can be used to generate a summary of the executed CLI command.
-->
This functionality can be used to generate a summary in the context of an entire workflow (a sequence of JFrog CLU commands) and not only in the scope of a specific command.